templates/systemd-service.template

```diff
@@ -19,7 +19,12 @@ RestartSec=2s
 
 ExecStartPre=-/usr/bin/docker stop {{ service_name }}
 ExecStartPre=-/usr/bin/docker rm {{ service_name }}
-ExecStartPre={{ service_pre_command }}
+
+{% if service_pre_command == "" %}
```

Comment:
Instead of checking if it's empty, check if it's defined or not with `if service_pre_command is defined`.

Reply:
>
check if it's defined or not
>
I think using a try block here is more suitable than an if block